README.md

```diff
@@ -871,18 +876,34 @@ class UserMailer < ApplicationMailer
     mail to: @user.confirmable_email, subject: "Confirmation Instructions"
   end
 end
+```
 
+3. Update Confirmations Controller.
+
+```ruby
+# README.md
+class ConfirmationsController < ApplicationController
+  ...
+  def edit
+    ...
+    if @user.present? && @user.unconfirmed_or_reconfirming? && @user.confirmation_token_has_not_expired?
```

Comment:
We technically don't need `@user.unconfirmed_or_reconfirming?` since that's accounted for in `User#confirm!` but I'm OK with keeping it.